st3/lsp_utils/__init__.py

```diff
@@ -1,2 +1,12 @@
-from .npm_client_handler import NpmClientHandler
-from .server_npm_resource import ServerNpmResource
+try:
+    from .v1 import ApiWrapper
+    from .v1 import NpmClientHandler
+except ImportError:
+    from .v0 import ApiWrapper  # type: ignore
+    from .v0 import NpmClientHandler
```

Comment:
Works with both old and new. Some plugins like vscode-html and vue need some more adjustments.

Reply:
What kind of adjustments? `LSP_html` is as bare as it gets so I'm surprised that it needs something extra.

Reply:
I got confused because I had not done `git pull` in a while for LSP-html.